python/auto_sip.blacklist

```diff
@@ -247,6 +247,7 @@ core/symbology-ng/qgssymbollayerregistry.sip
 core/symbology-ng/qgssymbollayerutils.sip
 core/symbology-ng/qgsvectorfieldsymbollayer.sip
 core/symbology-ng/qgsarrowsymbollayer.sip
+core/symbology-ng/qgsconcentriclegendsymbol.sip
```

Comment:
If you have [prepare-commit](https://github.com/qgis/QGIS/blob/master/.github/CONTRIBUTE.md#contributing-to-qgis) set up, this should all happen transparently